tabular/src/autogluon/tabular/trainer/auto_trainer.py

```diff
@@ -21,9 +21,13 @@ def get_models(self, hyperparameters, **kwargs):
                                  num_classes=num_classes, hyperparameters=hyperparameters, invalid_model_names=invalid_model_names, **kwargs)
 
     # TODO: rename to .fit for 0.1
-    def train(self, X_train, y_train, X_val=None, y_val=None, X_unlabeled=None, hyperparameter_tune=False, feature_prune=False, holdout_frac=0.1, hyperparameters=None, ag_args_fit=None, excluded_model_types=None, time_limit=None, **kwargs):
+    def train(self, X_train, y_train, X_val=None, y_val=None, X_unlabeled=None, hyperparameter_tune=False, feature_prune=False, holdout_frac=0.1, stack_ensemble_levels=0, hyperparameters=None, ag_args=None, ag_args_fit=None, ag_args_ensemble=None, excluded_model_types=None, time_limit=None, **kwargs):
+        for key in kwargs:
+            logger.warning(f'Warning: Unknown argument passed to `AutoTrainer.train()`. Argument: {key}')
```

Comment:
Consider removing `AutoTrainer.train()` from the warning message since most users won't know what that is.

Reply:
This error is impossible for a normal user to get since they can't pass kwargs to trainer.train(), this is more a developer warning.